ccc/go.mod

```diff
@@ -1,11 +1,45 @@
 module ccc
 
-go 1.15
+go 1.19
 
 require (
 	github.com/MichaelMure/go-term-markdown v0.1.4
 	github.com/gin-gonic/gin v1.8.1
+	gopkg.in/yaml.v3 v3.0.0-20210107192922-496545a6307b
 	nullprogram.com/x/optparse v1.0.0
 )
 
-require gopkg.in/yaml.v2 v2.4.0
+require (
```

Comment:
Please check, if it's possible to remove those

Reply:
The removal of the indirect requires would be necessary after every go command which manipulates the go.mod file.
See also https://go.dev/ref/mod#go-mod-file-require